local-setup/py/worker.py

```diff
@@ -160,7 +160,8 @@ def run_in_background(self, log_file: TextIO, flags: [str] = None, subcommand_fl
                                         'enclave_runtime=info,'
                                         'integritee_service=warn,'
                                         'ita_stf=debug')
-
+        worker_cmd = self._assemble_cmd(flags=flags, subcommand_flags=subcommand_flags)
+        print("worker command is "+ str(worker_cmd))
         return Popen(
             self._assemble_cmd(flags=flags, subcommand_flags=subcommand_flags),
```

Comment:
we can replace the second `self._assemble_cmd` with the variable.

Reply:
Nice catch, I added this just for easier debugging, but yeah, this can stay! :100: